app/boards/intel_adsp_ace30_ptl.conf

```diff
@@ -10,7 +10,7 @@ CONFIG_COMP_SRC=y
 CONFIG_MM_DRV=y
 CONFIG_COMP_ARIA=y
 CONFIG_COMP_CHAIN_DMA=y
-CONFIG_COMP_DRC=y
+CONFIG_COMP_DRC=m
```

Comment:
I think we want to move all supported components to "m" for PTL, we need CI support for the "openmodules" library for that

Reply:
@lyakh are we ready to do this now or still WIP ?